### ISSUE TITLE
TSCONFIG: Error Key: 'target' has an invalid value: es2017

### ISSUE THREAD

**Issue by mavericken** [alm-tools/alm#395]
ALM does not like target:"es2017" in my tsconfig for some reason.

```
C:\Work\experimental\jspmreacttest>npm ls -g alm
C:\Users\Rick\AppData\Roaming\npm
`-- alm@2.23.0


C:\Work\experimental\jspmreacttest>npm ls -g typescript
C:\Users\Rick\AppData\Roaming\npm
+-- alm@2.23.0
| `-- typescript@2.2.0-dev.20161229
+-- typescript@2.1.5
`-- typings@1.3.0
  `-- typings-core@1.2.2
    `-- typescript@1.8.7


C:\Work\experimental\jspmreacttest>tsc --version
Version 2.1.5

C:\Work\experimental\jspmreacttest>alm
Version: 2.23.0
DASHBOARD: (Please open in chrome) http://localhost:4444
TSCONFIG: Error Key: 'target' has an invalid value: ES2017
```

```
{
  "compilerOptions": {   
    "baseUrl": ".",
    "rootDir": ".",
    "noImplicitAny": false,
    "noEmit": false,
    "module": "es2015",
    "target": "es2017",
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
    "jsx": "preserve"
  },
  "exclude": [
    "node_modules",
	  "jspm_packages"
  ],
  "filesGlob": [
    "*{.ts, .d.ts}",
    "typings/browser/**/*{.ts, .d.ts}",
    "typings/globals/**/*{.ts, .d.ts}",
    "typings/index.d.ts",
    "typings-manual/**/*{.ts, .d.ts}",
    "node_modules/mobx/lib/*{.ts, .d.ts}"
  ],
  "compileOnSave": false,
  "atom": {
    "rewriteTsconfig": true
  }
}
```

**Comment by mavericken** [comment]
Actually, I am using this version for vs-code, and vs-code doesn't mind es2017 setting:
```
C:\work\experimental\jspmreacttest\.vscode>npm ls typescript
C:\work\experimental\jspmreacttest
`-- typescript@2.2.0-dev.20161228
```
Its different by one day from the one ALM seems to be using.

**Comment by basarat** [comment]
Released with `2.24.0` :rose:
